code/modules/antagonists/morph/morph.dm

```diff
@@ -213,7 +213,9 @@
 	visible_message("<span class='warning'>[src] suddenly twists and changes shape, becoming a copy of [target]!</span>", \
 					"<span class='notice'>You twist your body and assume the form of [target].</span>")
 	appearance = target.appearance
-	copy_overlays(target)
+	copy_overlays(target, TRUE)
```

Comment:
```suggestion
```

I told you, `appearance` copying above also copies overlays. The thing works without this line. I even tested it.